libs/ui-watt/src/lib/foundations/color/storybook/color.stories.ts

```diff
@@ -0,0 +1,45 @@
+import { moduleMetadata, Story, Meta } from '@storybook/angular';
+import { MatTableModule } from '@angular/material/table';
+import { MatCardModule } from '@angular/material/card';
+
+import { ColorComponent } from './color.component';
+
+export default {
+  title: 'Foundations/Color',
+  component: ColorComponent,
+  decorators: [
+    moduleMetadata({
+      imports: [MatTableModule, MatCardModule],
+      providers: [
+        { provide: Window, useValue: window }
+      ]
+    })
```

Comment:
Suggestion: Consider creating a `ColorModule` module similar to `TypographyModule` and move these things there. It makes it easier to see how the Material modules are related to the component where they are used in and it's easier to reuse (if necessary).